bsp/boards/scum/radiotimer.c

```diff
@@ -0,0 +1,135 @@
+/**
+\brief SCuM-specific definition of the "radiotimer" bsp module.
+
+\author Tengfei Chang <tengfei.chang@inria.fr>, Auguest 2016.
+*/
+
+#include "memory_map.h"
+#include "stdio.h"
+#include "stdint.h"
+#include "string.h"
+#include "radiotimer.h"
+#include "board.h"
+#include "bsp_timer.h"
+
+//=========================== variables =======================================
+
+typedef struct {
+    radiotimer_compare_cbt    overflow_cb;
+    radiotimer_compare_cbt    compare_cb;
+    PORT_RADIOTIMER_WIDTH     interrupt_flag;
+} radiotimer_vars_t;
+
+radiotimer_vars_t radiotimer_vars;
+
+//=========================== prototypes ======================================
+
+//=========================== public ==========================================
+
+//===== admin
+
+void radiotimer_init() {
+    // clear local variables
+    memset(&radiotimer_vars,0,sizeof(radiotimer_vars_t));
+}
+
+void radiotimer_setOverflowCb(radiotimer_compare_cbt cb) {
+    radiotimer_vars.overflow_cb    = cb;
+}
+
+void radiotimer_setCompareCb(radiotimer_compare_cbt cb) {
+    radiotimer_vars.compare_cb     = cb;
+}
+
+void radiotimer_setStartFrameCb(radiotimer_capture_cbt cb) {
+    while(1);
+}
+
+void radiotimer_setEndFrameCb(radiotimer_capture_cbt cb) {
+    while(1);
+}
+
+void radiotimer_start(PORT_RADIOTIMER_WIDTH period) {
+    // set period of radiotimer
+    RFTIMER_REG__MAX_COUNT          = period*61/4;

```

Comment:
why the 61? please add comment
I would be very careful about using different operations on a single line. create a temp variable on which you do the operations, then apply to register